source/chess.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+/**
+ * Создание ASCII-шахматной доски размером boardSize*boardSize
+ * @param {number|string} boardSize - размер доски
+ * @returns {string|null} Шахматная доска
+ */
+const chess = (boardSize) => {
+    if (!(typeof boardSize) === "number" || !(typeof boardSize) === "string"
+        || boardSize <= 1 || boardSize % 1 != 0) return null
```

Comment:
1) Хочу создать доску с `boardSize = Infinity`. Кто сможет со мной совладать?
2) Упс! 

![изображение](https://github.com/frontend-park-mail-ru/homework_2023_2/assets/31134795/5afd937a-9f4c-4dbd-a0a1-8d4662f11341)

При этом это валидные строки... непорядок! Давай подумаем, как это починить)
3) Лучше использовать одиночные кавычки и вытащить отрицание в сам оператор:
```suggestion
    if (typeof boardSize !== 'number' || typeof boardSize !== 'string'
        || boardSize <= 1 || boardSize % 1 != 0) return null
```
4) Выглядит, как кейс для [ошибки](https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Global_Objects/Error), давай используем